src/app/modules/orders/order.ts

```diff
@@ -12,9 +13,10 @@ export interface Order {
 
 export interface MenuItem {
   _id?: string,
-  name: string,
+  name?: string,
   description?: string
   category?: Category
+  status?: PlateItemAction;
```

Comment:
è corretto a livello di menuItem o deve stare nell'order? L'order in realtà rappresenta l'associazione menuItem - ordine. Il menuItem rappresenta invece la singola voce di menù ma senza uno stato effettivo.